specification/liftrneon/Neon.Postgres.Management/tspconfig.yaml

```diff
@@ -27,3 +27,9 @@ options:
     flavor: "azure"
     generate-test: true
     generate-sample: true
+  "@azure-tools/typespec-java":
+    service-dir: "sdk/neonpostgres"
+    package-dir: "azure-resourcemanager-neonpostgres"
+    namespace: "com.azure.resourcemanager.neonpostgres"
+    service-name: "Neon Postgres"
+    flavor: "azure"
```

Comment:
add `examples-directory`?

Reply:
No longer needed. Default value is same.

Reply:
cool, I see, thanks

Reply:
Remove the parameter `service-dir`, leaving only basic parameters.